src/A1_csv_to_json.py

```diff
@@ -406,7 +411,7 @@ def create_json_from_csv(input_directory, filename, parameters):
         "simulation_settings",
     ]:
         return single_dict
-    elif "storage_" in filename:
+    elif storage == True:
```

Comment:
```suggestion
    elif storage == True:
```

Add description here (#... adding storage components)?

Reply:
```suggestion
    elif storage is True:
```